packages/whatsie-git/whatsie-git.pacscript

```diff
@@ -0,0 +1,19 @@
+name="whatsie-git"
+gives="whatsie"
+version="4.14.1"
```

Comment:
This should be using `pkgver()`. Consult the wiki and [other git packages](https://github.com/search?q=repo%3Apacstall%2Fpacstall-programs%20pkgver&type=code) on how to do this.

Reply:
I don't know much about shell, I got the list with the last output being the version I wanted with the following command: `git ls-remote "${url}" | grep -i "tags/v" | cut -d '/'  -f3 | cut -d 'v' -f2 | sort --version-sort`

Reply:
But then, it didn't work, returning the error:
```shell
[+] INFO: Packaging whatsie-git as whatsie
        [>] Packing control.tar
        [>] Packing data.tar
        [>] Compressing
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'whatsie:none' instead of '/usr/src/pacstall/whatsie-git.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 whatsie:none : Depends: qtbase5-dev:none but it is not installable
                Depends: qtdeclarative5-dev:none but it is not installable
                Depends: qtlocation5-dev:none but it is not installable
                Depends: libqt5webchannel5-dev:none but it is not installable
                Depends: qtwebengine5-dev:none but it is not installable
                Depends: qtwebengine5-private-dev:none but it is not installable
        [!] ERROR: Failed to install whatsie-git deb
[+] INFO: Cleaning up
```

Reply:
I thought that if I gave pacscript a list, it would get the first result automatically.

But that didn't happen, so he didn't know which version to use. Then, I looked for how to get the first line of a shell output and that's it, solved.

`git ls-remote "${url}" | grep -i "tags/v" | cut -d '/' -f3 | cut -d 'v' -f2 | sort -r --version-sort | head -n 1`